install_script.sh.template

```diff
@@ -1631,6 +1645,17 @@ elif [ ! "$no_agent" ]; then
     update_hostname "$sudo_cmd" "$hostname" "$config_file"
     update_hosttags "$sudo_cmd" "$host_tags" "$config_file"
     update_env "$sudo_cmd" "$dd_env" "$config_file"
+    if [ -n "$remote_updates" ]; then
```

Comment:
nit: moving this to its own function / in update_remote_updates would make it easier to read